pilot/pkg/model/config_test.go

```diff
@@ -582,6 +591,46 @@ func TestAuthenticationPolicyConfig(t *testing.T) {
 			},
 			PrincipalBinding: authn.PrincipalBinding_USE_ORIGIN,
 		},
+		"mockonce": {
+			Targets: []*authn.TargetSelector{{
+				Name: "mockonce",
+				Ports: []*authn.PortSelector{
+					{
+						Port: &authn.PortSelector_Number{
+							Number: 80,
+						},
+					},
+				},
+			}},
+			Origins: []*authn.OriginAuthenticationMethod{
+				{
+					Jwt: &authn.Jwt{
+						Issuer: "http://localhost:9999",
+					},
+				},
+			},
+			PrincipalBinding: authn.PrincipalBinding_USE_ORIGIN,
+		},
+		"mockoncetwice": {
```

Comment:
What's the different of this case with the one above?

Reply:
they're the same, just want to use them to verify mock server is only called once because of the cache.